app/Exceptions/ManifestDoesntExistException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Pterodactyl\Exceptions;
+
+use Exception;
+use Spatie\Ignition\Contracts\Solution;
+use Spatie\Ignition\Contracts\ProvidesSolution;
+
+class ManifestDoesntExistException extends Exception implements ProvidesSolution
+{
+    public function getSolution(): Solution
+    {
+        return new Solutions\ManifestDoesntExistSolution();
```

Comment:
Is there any specific reason this is a relative import rather than being fully imported?

Reply:
One less import and less characters. Honestly I should've done the same thing for Exception because it's only referenced once as the extended class.